recOrder/cli/apply_inverse_transfer_function.py

```diff
@@ -135,10 +97,8 @@ def apply_inverse_transfer_function_cli(
     tczyx_data = torch.tensor(input_dataset.data, dtype=torch.float32)
```

Comment:
when I ran this with the `uint16` datasets, I was getting an error that torch did not support that format and had to open `input_dataset.data.astype(np.float32)` instead.

Reply:
I noticed this when the datasets datatype when using `iohub info -v ./dataset.zarr`

Reply:
Did you run into this problem on this branch as well? I'm testing with `np.uint16` here, but you may have seen a different issue. If this is still an issue, can you point to an example dataset?

https://github.com/mehta-lab/recOrder/blob/ea9d4c976bdb2b5c019aec5741563ca14d1259e8/recOrder/tests/cli_tests/test_apply_inv_tf.py#L20